fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ShowStmtAnalyzer.java

```diff
@@ -331,6 +338,24 @@ public Void visitDescTableStmt(DescribeStmt node, ConnectContext context) {
             return null;
         }
 
+        private void descTableFunctionTable(DescribeStmt node, ConnectContext context) {
+            Table table = null;
+            try {
+                table = new TableFunctionTable(node.getTableFunctionProperties());
+            } catch (DdlException e) {
+                throw new StorageAccessException(e);
+            }
+
+            List<Column> columns = table.getFullSchema();
+            for (Column column : columns) {
+                List<String> row = Arrays.asList(
+                        column.getName(),
+                        column.getType().canonicalName().toLowerCase(),
```

Comment:
UPPER case?

Reply:
https://github.com/StarRocks/starrocks/blob/main/fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ShowStmtAnalyzer.java#L447

In MySQL, the Type column should lowercase, and the Null column should uppercase.
keep consistent with olap table?